interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is javascript in react. {} is required to write javascript inside of an html area
 
-Researched answer:
+Researched answer: JSX is what allows React to create user interfaces. JSX uses className while HTML uses class. this is because class already means something else inside javascript.
 
 2. What is yarn? What file(s) are modified in a React application when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: yarn is an application that is reached thru terminal. you get yarn.lock.
 
-Researched answer:
+Researched answer: A javascript testing framework. the files test.js node_modules, package.json and yarn.lock
```

Comment:
Jest is a Javascript testing framework, yarn is a package manager that will modify the node_modules, package.json and yarn.lock.